MinnPost/object-sync-for-salesforce
# Fix the API Name settings so the value shows up correctly. Thanks to WordPress user @dcleslie777 for the report.

## What does this Pull Request do?

This fixes #392. When API Name was selected in the plugin settings, it was not displaying the value correctly on new fieldmap rows. This fixes the script so it respects the setting and loads the correct data.

## How do I test this Pull Request?

- Set API Name as the value for Salesforce Field Display Value on the plugin settings.
- Create a new fieldmap
- Check the Salesforce field dropdown
